src/components/Form/Form.tsx

```diff
@@ -58,7 +57,12 @@ const Form = <T extends FormDetails>({
     event.stopPropagation();
 
     if (validate) {
-      const errors = omitBy(validate(state.form), isEmpty) as FormErrors<T>;
+      const errors = Object.fromEntries(
```

Comment:
I think this should be abstracted into a helper function

Reply:
@gigincg  ok so I am making useOmitBy.ts in /hooks and then using it here
is it ok
because I made useDebounce.ts for debounce

sry i didn't catch this one